src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/FormattingTestBase.cs

```diff
@@ -158,6 +149,22 @@ private protected async Task RunOnTypeFormattingTestAsync(
         }
     }
 
+    private (TestCode, string) ProcessFormattingContext(TestCode input, string expected)
+    {
+        Assert.True(_context.CreatedByFormattingDiscoverer, "Test class is using FormattingTestContext, but not using [FormattingTestFact] or [FormattingTestTheory]");
+
+        UpdateClientInitializationOptions(opt => opt with { ForceRuntimeCodeGeneration = _context.ForceRuntimeCodeGeneration });
```

Comment:
Does this default to true in the context?

Reply:
Strictly speaking the default is false, but it's also never used. The test runner creates multiple test cases for each combination, so even the facts act like theories. See here: https://github.com/dotnet/razor/blob/main/src/Shared/Microsoft.AspNetCore.Razor.Test.Common/Formatting/FormattingFactDiscoverer.cs#L20-L21

Reply:
Ah, that makes sense